test/Microsoft.AspNet.Mvc.Razor.Host.Test/MvcRazorParserTest.cs

```diff
@@ -15,30 +16,132 @@ namespace Microsoft.AspNet.Mvc.Razor
 {
     public class MvcRazorCodeParserTest
```

Comment:
That's an invalid case, for the complexity of this test it'll work but in reality it will not; didn't want to add something that'll be invalid in the real world.